CMakeLists.txt

```diff
@@ -79,6 +79,7 @@ IF (KokkosKernels_INSTALL_TESTING)
   KOKKOSKERNELS_ADD_TEST_DIRECTORIES(test_common)
   KOKKOSKERNELS_ADD_TEST_DIRECTORIES(perf_test)
   KOKKOSKERNELS_ADD_TEST_DIRECTORIES(unit_test)
+  KOKKOSKERNELS_ADD_TEST_DIRECTORIES(smoke_test)
```

Comment:
I think we'd probably want this folder always enabled, not just with tests. Most Spack builds won't enable tests

Reply:
I somewhat dislike the idea of always installing files that users do not need/want when they do a CMake build, I am considering enabling this directory based on a CMake like: `-KokkosKernels_ENABLE_SMOKE_TEST` I think it would be a little cleaner?

Reply:
Yeah, that's better. We can do some Spack foo to add that option.